portfoliostyle.css

```diff
@@ -0,0 +1,310 @@
+	margin-top: 0px;
+}
+.about img {
+	object-fit: scale-down;
+	margin-right: 25px;
+	max-width: 40%;
+}
+
+.about-projects {
+	display: flex;
+	flex-direction: column;
+}
+
+.projects {
+	display: flex;
+	flex-direction: column;
+	margin-top: 25px;
+	padding: 25px;
+	background-color: #464866;
+	color: #ffffff;
+	border-radius: 10px;
+}
+
+.project-container {
+	flex-direction: row;
+	justify-content: space-around;
+}
+
+.project-container div {
+	flex-direction: column;
+	align-items: center;
+}
+
+h3 {
+	margin-bottom: 0px;
+	margin-top: 0px;
+}
+
+.projects img {
+	width: 100px;
+	height: 100px;
+	margin: 10px 0 10px 0;
+}
+
+.projects button {
+	padding: 10px;
+	border-radius: 10px;
+	background-color: #29648a;
+	color: #ffffff;
+	border-color: #2e9cca;
+}
+
+
+.contacts {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	margin-top: 25px;
+	padding: 0 25px 0 25px;
+	background-color: #464866;
+	color: #ffffff;
+	border-radius: 10px;
+	align-self: center;
+	width: 90%;
+}
+
+.contacts-icons {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	margin-top: 10px;
+	padding: 0 25px 0 25px;
+	border-radius: 10px;
+}
+
+.fa {
+	padding: 20px;
+ 	font-size: 30px;
+	text-align: center;
+  	text-decoration: none;
+  	border-radius: 10%;
+  	margin: 5px;
+}
+
+.fa-envelope-square {
+  	background: #dd4b39;
+	color: #ffffff;
+}
+
+.fa-linkedin {
+  	background: #007bb5;
+  	color: #ffffff;
+}
+
+.fa-phone {
+	background: green;
+	color: #ffffff;
+}
+
+.fa-github-square {
+	background: #000000;
+	color: #ffffff;
+}
+
+.fa:hover {
+  	opacity: 0.7;
+}
+
+button:hover {
+    opacity: 0.7;
+}
+
+
+/*-----------------Media Queries---------------------*/
+
+@media screen and (max-width: 850px) {
+	.project-container {
+		flex-direction: column;
+		justify-content: space-between;
+	}
+}
+
+
+@media screen and (max-width: 850px) {
+	.about div {
+		flex-direction: column;
+	}
+}
+
+
+@media screen and (max-width: 850px) {
+	.about-projects {
+		flex-direction: row;
+		flex-wrap: wrap;
+		justify-content: space-around;
+	}
+}
+
+@media screen and (max-width: 850px) {
+	.about {
+		flex-direction: column;
+		/*width: 350px;*/
+		width: 40%;
+		font-size: 2.4vw;
+	}
+}
+
+
+@media screen and (max-width: 850px) {
+	.projects {
+		flex-direction: column;
+		/*width: 350px;*/
+		width: 40%;
+		font-size: 2.4vw;
+	}
+}
+
+
+@media screen and (max-width: 850px) {
+	.contacts {
+		width: 90%;
+	}
+}
+
+@media screen and (max-width: 850px) {
+	.about img {
+		object-fit: scale-down;
+		max-width: 100%;
+	}
+}
+
+
+
+
+@media screen and (max-width: 700px) {
+	.about-projects {
+		flex-direction: column;
+		align-items: center;
+
+	}
+}
+
+@media screen and (max-width: 700px) {
+	.contacts {
+		width: 40%;
+	}
+}
+
+@media screen and (max-width: 700px) {
+	.about img {
+		object-fit: scale-down;
+		max-width: 100%;
+	}
+}
+
+@media screen and (max-width: 700px) {
+	.project-container img {
+		/*object-fit: scale-down;
```

Comment:
Can remove comment if you no longer need this code :)

Reply:
Removed